examples/example.js

```diff
@@ -464,15 +502,15 @@ async function scheduleERC20TransferJob(owner, token, taskCondition) {
     metadata
   );
 
-  console.log("Expression Task ID is:", result);
+  return result;
```

Comment:
Currently, edges return: 
```
edges: [ { id: 'edge-123abcdef', start: 'transfer_erc20_1', end: '' } ],
```

In React Flow, the edges are returned as an array of objects, with the following properties:
```
  id: string;
  source: string;
  target: string;
  type: string; (not useful to backend)
  data: object; (not useful to backend, yet)
```

If you are not against it, let’s use `{id, source, target}` for the edges, so the property names look a little more sophiscated, and we don’t need an extra converter between the two formats.